elasticjob-lite/elasticjob-lite-spring/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/setup/JobClassNameBySpringIdentificationStrategy.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.elasticjob.lite.spring.setup;
+
+import org.apache.shardingsphere.elasticjob.api.ElasticJob;
+import org.apache.shardingsphere.elasticjob.lite.internal.setup.JobIdentificationStrategy;
+import org.apache.shardingsphere.elasticjob.lite.spring.job.util.AopTargetUtils;
+import org.springframework.aop.support.AopUtils;
+
+/**
+ * Identify job with job class name.
+ * <p>
+ * Consider the proxy object that generated by cglib or jdk dynamic proxy.
+ * </p>
+ */
+public class JobClassNameBySpringIdentificationStrategy implements JobIdentificationStrategy {
+
+    @Override
+    public String identify(final ElasticJob elasticJob, final String jobName) {
```

Comment:
Should return Optional for next JobIdentificationStrategy, just like response chain

Reply:
Thank you very much, I'll finish it in the next PR.